mobile/cmp/navigator/impl/Page.ts

```diff
@@ -26,22 +25,16 @@ export const page = hoistCmp.factory({
     model: uses(PageModel, {publishMode: 'limited'}),
 
     render({model}) {
-        const {content, props, isActive, renderMode, refreshContextModel} = model,
-            wasActivated = useRef(false);
+        const {content, props, isActive, renderMode, refreshContextModel} = model;
 
-        if (!wasActivated.current && isActive) wasActivated.current = true;
-
```

Comment:
This is removed because we are always lazy now?

Reply:
We were always lazy before too! Good idea to throw on 'always', will add that now.